### PR TITLE
dougiesquire/issue201

### LINKED ISSUE #201: Add additional resources to MED section
Some additional resources to add to Model Evaluation and Diagnostics:

- [xskillscore](https://xskillscore.readthedocs.io/en/stable/#) : a Python package for computing a wide variety of skill metrics
- [climpred](https://climpred.readthedocs.io/en/stable/) : a Python package for verifying weather and climate forecasts
- [GCMeval](https://gcmeval.met.no/) : a web application for selection and evaluation of climate model data. This doesn't appear to have received and updates since [late 2020](https://github.com/metno/gcmeval/) so maybe not worth including?
- [MONET](https://github.com/noaa-oar-arl/monet) : toolkit for evaluation of atmospheric composition data
- [GeoCAT](https://geocat.ucar.edu/) : the new (Python-based) NCL. Probably a bit tangential, but they recently reached out to `xskillscore` devs to ask about using us as a dependency, so perhaps GeoCAT will provide evaluation metrics at some point?

Also, note to self to add link to [MET paper](https://journals.ametsoc.org/view/journals/bams/102/4/BAMS-D-19-0093.1.xml) to METplus section

### PR DESCRIPTION
**Pull request by dougiesquire**
# ACCESS Community Hive 

## Description

Adds a few additional resources to the MED section as described in the issue below

Fixes #201 

## Type of change

Please delete options that are not relevant.

- [x] New link / content

## Checklist:

- [x] The new content is accessible and located in the appropriate section.
- [x] My changes do not break navigation and do not generate new warnings
- [x] I have checked that the links are valid and point to the intended content.
- [x] I have checked my code/text and corrected any misspellings
- [x] I have chosen the correct tag for the level of [support provided](https://access-hive.org.au/#support).

### CODE REVIEW

**Comment by dougiesquire** [comment]
Note, I decided not to add [GCMeval](https://gcmeval.met.no/) or [GeoCAT](https://geocat.ucar.edu/), for reasons explained in #201

File: docs/model_evaluation/atmosphere.md [hunk 1]
```diff
@@ -11,9 +12,11 @@
 
 ESMValTool is a community-developed climate model diagnostics and evaluation software package, driven both by computational performance and scientific accuracy and reproducibility. ESMValTool is open to both users and developers, encouraging open exchange of diagnostic source code and evaluation results from the Coupled Model Intercomparison Project CMIP ensemble. For a comprehensive introduction to ESMValTool please visit our documentation page.
 
+
 ## [METPLUS][metplus-web]  {{ recommended }}
 
-[**Tutorial**][metplus-tutorial]
+[**Tutorial**][metplus-tutorial] |
+[**Paper**][MET-paper]
```
Comment (rbeucher) [inline docs/model_evaluation/atmosphere.md:19]: I have added the option to cite publications that we store in the bib file.
At some point we will have a list of papers of interest. 
Have a look at the model components for examples.

File: docs/model_evaluation/atmosphere.md [hunk 2]
```diff
@@ -57,10 +61,33 @@ Freva, the free evaluation system framework, is a data search and analysis platf
...
-
 TECA is a general purpose tool for detecting discrete events in climate model output. It leverages a map-reduce framework for efficient parallelization at large scales (order 10K+ cores). Currently, TECA contains detection algorithms for tropical cyclones, atmospheric rivers, and extratropical cyclones; and plans are underway to implement algorithms for mesoscale convective complexes, African Easterly waves, atmospheric blocks, and fronts.
 
 
+## [Model and ObservatioN Evaluation Toolkit (MONET)][monet-doc]  {{ community }}
+
+[**Documentation**][monet-doc] |
+[**Tutorial**][monet-tutorial] |
+[**Source Code**][monet-source] |
+[**Paper**][monet-paper]
```
Comment (rbeucher) [inline docs/model_evaluation/atmosphere.md:72]: Same here

File: docs/model_evaluation/atmosphere.md [hunk 3]
```diff
@@ -57,10 +61,33 @@ Freva, the free evaluation system framework, is a data search and analysis platf
...
+
+MONET was developed to evaluate the Community Multiscale Air Quality Model (CMAQ) for the NOAA National Air Quality Forecast Capability (NAQFC) modeling system. MONET is designed to be a modularized Python package for (1) pairing model output to observational data in space and time; (2) leveraging the Pandas Python package for easy searching and grouping; and (3) analyzing and visualizing data. This process introduces a convenient method for evaluating model output.
+
+
+## [climpred][climpred-doc]  {{ community }}
+
+[**Documentation**][climpred-doc] |
+[**Tutorial**][climpred-tutorial] |
+[**Source Code**][climpred-source] |
+[**Paper**][climpred-paper]
```
Comment (rbeucher) [inline docs/model_evaluation/atmosphere.md:84]: Same here

**Review by rbeucher — COMMENTED** [review]
Looks good @dougiesquire .
See my comments about citing the papers.

**Review by rbeucher — APPROVED** [review]

**Pull request merged**